core/orchestrator.go

```diff
@@ -390,8 +390,9 @@ type TranscodeData struct {
 
 // TranscodedSegmentData contains encoded data for a profile
 type TranscodedSegmentData struct {
-	Data   []byte
-	Pixels int64 // Encoded pixels
+	Data    []byte
+	SigData []byte // Perceptual signature data (maybe nil)
```

Comment:
```suggestion
	PHash []byte // Perceptual hash data (maybe nil)
```

AFAIK "perceptual signature" isn't a standard term so if the goal here is to use general terms that capture the category of algorithms that MPEG-7 video signatures fall into then I suggest using the term "perceptual hash" as that is a standard term in industry/research.